.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Something's not working correctly
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+*Example: I cannot add a tag when clicking on its name.*
```

Comment:
Is this section really necessary? It looks more like instructions for the title of the issue.

Maybe we can rephrase like this:
> For the title, please write a clear and concise description of what the bug is.
> *Example: I cannot add a tag when clicking on its name.*